edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -151,26 +151,25 @@ <h4>
     }, failureCallback);
   }
 
-  function launchSystemCheck() {
-    setPrimacyOfButtons(false);
-    var url = $('.instructions').data('exam-started-poll-url');
-    var action = $('#software_download_link').data('action');
-    // open the new tab in the click event with an empty URL but show the message.
-    var newWindow = window.open("", "_blank");
-    $(newWindow.document.body).html("<p>Please wait while you are being redirected...</p>");
-
-    $.ajax({
-      url: url,
-      type: 'PUT',
-      data: {
-        action: action
-      },
-      success: function (data) {
-        newWindow.location = "{{software_download_url|safe}}";
-      }
-    }).fail(function(){
-      newWindow.close();
-    });
+ function launchSystemCheck() {
+   setPrimacyOfButtons(false);
+   var url = $('.instructions').data('exam-started-poll-url');
+   var action = $('#software_download_link').data('action');
+   // open the new tab in the click event with an empty URL but show the message.
+   var newWindow = window.open("{{software_download_url|safe}}", "_blank");
+   if (!newWindow) {
+     edx.courseware.proctored_exam.accessibleError(
+       gettext("Please Allow Pop-ups"),
+       gettext('Allow pop-ups for this website and click "Start System Check" again to proceed.')
```

Comment:
Following up on my [comment](https://openedx.atlassian.net/browse/EDUCATOR-4242?focusedCommentId=374077&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-374077) on the JIRA ticket, maybe something like this would work. What do you think?

`Please allow pop-ups in your browser settings and/or in any browser extensions and then click "Start System Check" again to proceed.`

Reply:
Edited slightly to make less repetitive